detection-rules/link_download_suspicious_file.yml

```diff
@@ -14,7 +16,16 @@ severity: "medium"
 source: |
   type.inbound
   and any(body.links,
-          any(ml.link_analysis(.).files_downloaded,
+          // Detect suspicious direct Google Drive downloads
+          (
+            strings.icontains(.href_url.url, "drive.google.com/uc") 
+            and strings.icontains(.href_url.url, "export=download")
+            and any(ml.link_analysis(., mode="aggressive").files_downloaded,
+                .file_extension in $file_extensions_common_archives
```

Comment:
Do you want to add any file.explode to that downloaded archive to inspect the file extension of the filed within the arcive?

Reply:
Unless we see FPs, not really. A direct download link to an archive is weird enough IMO